docs/en/getting-started/install.md

```diff
@@ -346,8 +350,31 @@ You’ll need to create data and metadata folders manually and `chown` them for
 
 On Gentoo, you can just use `emerge clickhouse` to install ClickHouse from sources.
 
-### From CI checks pre-built binaries
-ClickHouse binaries are built for each [commit](/docs/en/development/build.md#you-dont-have-to-build-clickhouse).
+### Install a CI-generated Binary
+
```

Comment:
Are you going to remove https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/development/build.md#you-dont-have-to-build-clickhouse-you-dont-have-to-build-clickhouse ? If this content needs to be in both places let me know and I will make it a snippet that can be single sourced.

Reply:
Good point. Guess I rather merge the existing instructions with the new ones. Let me make this more consistent.